.circleci/config.yml

```diff
@@ -15,15 +15,30 @@ workflows:
       - backend_tests:
           requires:
             - setup
+      - e2e_admin:
```

Comment:
For future ref: let's please add documentation here in a follow-up PR to explain the difference between prod/non-prod and the intended blocking structure of the tests (maybe follow the outline given below, or whatever we end up deciding). Also, explain how to tell if an e2e test is non-prod.

- Setup runs at first.
- After setup completes, lint and frontend tests (05:02) run. If frontend tests fail, stop.
- After frontend tests complete, backend (30:25) and typescript (01:01) tests run. If typescript tests fail, stop.
- After typescript tests complete, the flagship e2e tests (navigation 09:21 and file upload 02:35) run.
- After file upload e2e test completes, all other non-prod e2e tests run in parallel.
- After navigation e2e test completes, all other prod e2e tests run in parallel.